<launch>
  <arg name="path_to_gt_map" default="/home/kirill/TopoSLAM/GT/mipt_map"/>
  <arg name="path_to_load_json" default="/home/kirill/TopoSLAM/toposlam_ws/data/habitat_mipt_bags/mipt_rlnav"/>
  <arg name="config_file" default="habitat_mipt_noised_odom.yaml"/>

  <node pkg="prism_topomap" type="prism_topomap_node.py" name="prism_topomap_node" output="screen">
    <param name="path_to_gt_map" value="$(arg path_to_gt_map)"/>
    <param name="path_to_load_graph" value="$(arg path_to_load_json)"/>
    <param name="config_file" value="$(arg config_file)"/>
  </node>

  <node pkg="tf" type="static_transform_publisher" name="tf_map_to_odom" args="0 0 0 0 0 0 map odom 100"/>
</launch>